python/setup.py

```diff
@@ -92,11 +92,11 @@ def run(self):
     install_requires = [
         "numpy",
         "typing",
-        "pandas == 1.1.4",
+        "pandas >= 1.1.4",
         "psutil",
         "pyarrow >= 0.10",
-        "ray == 1.4.0",
-        "pyspark >= 3.0.0, < 3.1.0",
+        "ray >= 1.4.0, < 1.5.0",
```

Comment:
@carsonwang  - would this be alright, allowing patch changes?

Reply:
Yes, looks good. Thanks!